.eslintrc.js

```diff
@@ -4,8 +4,7 @@ module.exports = {
     "react-app/jest",
   ],
   rules: {
-    // Currently 120 warnings.
-    "@typescript-eslint/no-explicit-any": "off",
+
   },
```

Comment:
Does it have to be empty, or can we remove that?

Reply:
Can remove that.